### DISCUSSION TITLE
TreeTable - scrollHeight responsive

### DISCUSSION THREAD

**Discussion by TeoDar** [primefaces/community#63] [PrimeVue] [answered]
Good day to all. I transfer the question from the forum (https://forum.primefaces.org/viewtopic.php?t=81064).
I have encountered the same problem, but in PrimeVue, and would like to know how to solve it.

Here's an [example](https://stackblitz.com/edit/jzzpv4?file=src%2FApp.vue).
I commented out the code that tries to change scrollHeight based on the window resizing event, because it still didn’t work.
But that's what I'm trying to achieve. Adaptive scrollHeight changes based on window resizing.

P.S. Maybe you don’t need to recalculate it at all, but set flex and it will recalculate itself?)

**Comment by cetincakiroglu** [comment]
Hi, 

Could you please share a Stackblitz example so we can play around to make it work?

**Reply by TeoDar** [reply]
Here's an [example](https://stackblitz.com/edit/jzzpv4?file=src%2FApp.vue).
I commented out the code that tries to change scrollHeight based on the window resizing event, because it still didn’t work.
But that's what I'm trying to achieve. Adaptive scrollHeight changes based on window resizing.

**Reply by cetincakiroglu** [reply]
This is a PrimeVue example, is the issue related to PrimeVue?

**Reply by TeoDar** [reply]
Sorry. I change to PrimeVue.
Initially, the issue on the forum was in PrimeNG, but the problem is the same for any of PrimeVue, Angular or React.

**Reply by cetincakiroglu** [reply]
Are you asking for help with primeng or primevue?

**Reply by TeoDar** [reply]
About PrimeVue.

**Reply by cetincakiroglu** [reply]
Could you please open this discussion in PrimeVue with the example so the PrimeVue team can help you easily?

**Reply by cetincakiroglu** [reply]
For PrimeNG issue of this topic, we're aware of the issue and right now we're fully focused on accessibility implementation. So the issue is currently pending state. We'll work on fixing it after the accessibility implementation is done.

**Reply by TeoDar** [reply]
I'm already moved issue in PrimeVue section. Thanks for the explanation)

**Comment by TeoDar** [comment, marked as answer]
Wow. I just changed my [example](https://stackblitz.com/edit/jzzpv4?file=src%2FApp.vue), keeping the flex parent, set scrollHeight for TreeTable=flex and it helped. The scrollHeight itself is recalculated when the window is resized.

Inspired by the example [two issues below](https://github.com/orgs/primefaces/discussions/36).

I did not find such an explanation in the documentation, and I hope this will help someone in the future🤗

**Reply by TeoDar** [reply]
By the way. In my personal example (I can’t post it here), this didn’t help much either, but I set overflow: hidden. And after that, TreeTable stopped going beyond the boundaries of the parent Card.
